package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1035,11 +1035,13 @@ def principal_axes(group, pbc=False):
 
 
         .. versionchanged:: 0.8 Added *pbc* keyword
+        .. versionchanged:: 1.0.0 
```

Comment:
Also add text to the doc string of the method saying that the eigenvectors form a right-handed coordinate system.

Reply:
Sorry, must've missed this. Now added.